src/Robo/Blt.php

```diff
@@ -116,6 +120,60 @@ private function addPluginsCommandsAndHooks() {
   }
 
   /**
+   * Adds Symfony (non-Robo) command classes to the application.
+   *
+   * @param \Acquia\Blt\Robo\Application $application
+   */
+  protected function addSymfonyCommands(Application $application) {
+    $twig = new Environment(new FilesystemLoader());
+
+    // Get any custom defined Twig filters to be ignored by linter.
+    $twig_filters = (array) $this->getConfig()->get('validate.twig.filters');
+    // Add Twig filters from Drupal TwigExtension to be ignored.
+    // @todo Find a way so that this list doesn't need to be maintained.
+    $drupal_filters = [
```

Comment:
Nobody loves regular expressions, but you could load the text content of `docroot/core/lib/Drupal/Core/Template/TwigExtension.php` and then use a regex to identify the functions and filters:

`$found = preg_match_all("#new \\Twig_SimpleFunction\('([^']+)',#", file_get_contents('docroot/core/lib/Drupal/Core/Template/TwigExtension.php'), $matches);`

Reply:
Oh, that's not a bad idea. I'll try to update the PR soon.

Reply:
Made the updates, and tested against canary.
I'm not sure why Travis is failing, though.